doc/sphinx/addendum/extraction.rst

```diff
@@ -32,26 +32,23 @@ Generating ML Code
   can be used to refer to any kind of Coq global "object" : :term:`constant`,
   inductive type, inductive constructor or module name.
 
-The next two commands are meant to be used for rapid preview of
-extraction. They both display extracted term(s) inside Coq.
-
 .. cmd:: Extraction @qualid
-
-   Extraction of the mentioned object in the Coq toplevel.
-
-.. cmd:: Recursive Extraction {+ @qualid }
-
-   Recursive extraction of all the mentioned objects and
-   all their dependencies in the Coq toplevel.
-
-All the following commands produce real ML files. User can choose to
-produce one monolithic file or one file per Coq library.
-
-.. cmd:: Extraction @string {+ @qualid }
-
-   Recursive extraction of all the mentioned objects and all
-   their dependencies in one monolithic file :token:`string`.
-   Global and local identifiers are renamed according to the chosen ML
+         Recursive Extraction {+ @qualid }
+         Extraction @string {+ @qualid }
+
+   The first two commands are meant to be used for rapid preview of
+   extraction. They both display extracted term(s) inside Coq:
+
+   - the first one extracts :n:`@qualid` and displays the resulting term;
+   - the second one extracts the listed :n:`@qualid`\s and all their
+     dependencies, and displays the resulting terms.
+
+   The third command, as well as the ones presented below, produce
+   extracted file(s) on the filesystem. This third command is used to
+   create a single monolithic file, whose name is given by :n:`@string`.
+   It will recursively extract all the mentioned objects and all
+   their dependencies.
+   Global and local identifiers are renamed according to the chosen
    language to fulfill its syntactic conventions, keeping original
    names as much as possible.
```

Comment:
```suggestion
   The third form produces a single extraction file named :n:`@string`
   for all the specified objects and all of their dependencies..

   Global and local identifiers are renamed as needed to fulfill the syntactic
   requirements of the target language, keeping original
   names as much as possible.
   
The following commands also generate file(s).  ADD STUFF HERE.
```
The renaming sentence applies to all the commands, so it could appear before all the commands.

If you know, perhaps you could say what directory the extraction files go into?  Current or the same directory containing `ident.v` (latter seems a poor choice)?  The created subdirectories, if any, follow the module structure or is it flat hierarchy?

Also, FWIW, I think the commands should have used`Extract` (a verb) rather than `Extraction` (a noun).

Reply:
> If you know, perhaps you could say what directory the extraction files go into? Current or the same directory containing `ident.v` (latter seems a poor choice)? The created subdirectories, if any, follow the module structure or is it flat hierarchy?

I will do this in another PR, because this is related to the `Cd` and `Pwd` commands, which should be moved to this chapter.